doc/containers.md

```diff
@@ -479,6 +479,10 @@ Disk entries are essentially mountpoints inside the container. They can
 either be a bind-mount of an existing file or directory on the host, or
 if the source is a block device, a regular mount.
 
+LXD supports the following additional source types:
+- [ceph-rbd](#sourcetype-ceph-rbd): Mount from existing ceph RBD device that is externally managed
```

Comment:
Normal capitalization according to upstream is `Ceph`. I know we've been bad at doing this right, but it'd be great if this commit could use the correct capitalization :)